dtorrent/README.md

```diff
@@ -5,8 +5,8 @@ To run the program there needs to be a `config.cfg` file in the root of the proj
 Then run the program with `cargo` followed by the directory containing the .torrent files:
 
 ```bash
-$ cargo run --bin dtorrent ./torrents
-```
+$ cargo run --bin dtorrent ./torrents/file_name 
+```                                     
```

Comment:
Notice there are trailing whitespaces. This isn't anything to worry about, but it can cause the file to be modified by editors which remove trailing whitespaces automatically (like my own).

If you're using vscode, adding the following settings to the config should do that automatically for you (both with trailing spaces and newlines):
```json
  "files.trimFinalNewlines": true,
  "files.trimTrailingWhitespace": true,
  "files.insertFinalNewline": true,
```